docs/advanced-features/react-18/streaming.md

```diff
@@ -0,0 +1,62 @@
+# Streaming SSR (Alpha)
+
+React 18 will include architectural improvements to React server-side rendering (SSR) performance. This means you could use `Suspense` in your React components in streaming SSR mode and React will send render them and send through HTTP streams. React server components is also an experimental feature based on streaming. Next.js will also provide server components related features along with other streaming support through `next/streaming`.
```

Comment:
I'm wondering if the mention of server components above could be confusing (in that this doc does not address server components). Tried to clarify a bit:

```suggestion
React 18 will include architectural improvements to React server-side rendering (SSR) performance. This means you can use `Suspense` in your React components in streaming SSR mode and React will render them on the server and send them through HTTP streams. 

It's worth noting that another experimental feature, React Server Components, is based on streaming.  You can read more about server components related streaming APIs like `next/streaming` [here](). However, this guide focuses on basic React 18 streaming.
```